docs/user-guide/doc-odm-user-guide/studybrowser.md

```diff
@@ -1,84 +1,112 @@
+<figcaption>Search for data. Extend the query to include child terms in the search results. For example, the number of available studies with the word <strong>cancer</strong> increases when the option Extend query is selected</figcaption>
 
-If your search term matches a term from an ontology which has child terms, then once initial results have been retrieved there is the option to toggle ‘extend query’, which will add search results for child terms (those with the subClassOf property in the ontology). This is only available if the number of terms (including synonyms) is fewer than 30,000.
+## Filter Data
 
-### Filter panel
+* By default, all studies you have permission to access are displayed. Given the potential number of studies, search and filtering options in the left-hand panel of the **Study Browser** help you find specific studies of interest. 
+* You can search by study name, accession number, or any relevant metadata field. 
+* The facets beneath the search bar allow you to filter search results based on sample data (metadata) fields.
 
-The filter panel allows you to filter your results with search facets. Set facets to refine search results. Facets are
-automatically generated from the metadata of available datasets, and the suggested list of facets varies
-depending on a search results. Click ‘More’ to explore the whole list of suggested items.
+![10DataClass.png](doc-odm-user-guide/images/10DataClass.png)
+<figcaption>Filter data. Use the filters to narrow the search of studies. For example, select <strong>Data class</strong> and then tick the option <strong>Expression</strong> to find exclusively studies containing gene expression as experimental data (GCT format files)</figcaption>
 
-![image](doc-odm-user-guide/images/studybrowser_searchpanel.png)
+## Bookmark studies
 
-Bookmarked studies can be shown by clicking on the Bookmarks icon. Access allows you to see your studies, those that are accessible to you (for example, public studies) or those that have been shared with your user account/group.
+* You can save preferred studies for easy access. To the left of a study title is a three-dot link. Click this to open a menu that allows you to share the study (if you are the owner) and save it to bookmarks.
 
-If there are additional facet terms a **Find more** link will be displayed.
-Clicking on this allows you to type in terms and you can select from the presented list of options.
+![11Bookmarks.png](doc-odm-user-guide/images/11Bookmarks.png)
+<figcaption>Bookmark studies. Save preferred studies to access them easily. To do so, click on the three-dot link next to the name of the study and click on <strong>Save to bookmark</strong></figcaption>
 
-![image](doc-odm-user-guide/images/studybrowser_autocomplete.png)
+* Bookmarked studies are accessible by clicking on the Bookmarks icon. This section allows you to see your studies, those accessible to you (e.g., public studies), or those shared with your user account/group.
```

Comment:
Could you please explain what public studies are? Users can only see their own studies or those shared within the groups they belong to. Studies cannot be shared with individual users; they can only be shared with groups.